src/main/java/uk/gov/ons/census/fwmt/events/component/GatewayEventManager.java

```diff
@@ -21,9 +21,9 @@ public void addEventTypes(String[] et) {
     }
   }
 
-  public void triggerEvent(String caseId, String eventType) throws GatewayException {
+  public void triggerEvent(String caseId, String eventType, String eventTime) throws GatewayException {
     if (eventTypes.contains(eventType)) {
-      gatewayEventProducer.sendEvent(caseId, eventType);
+      gatewayEventProducer.sendEvent(caseId, eventType, eventTime);
```

Comment:
why isnt eventTime a date time  type field?

Reply:
Because it freaks out when it tries to map the DTO to an object